bot.js

```diff
@@ -65,11 +65,11 @@ function executeCommands (sender, to, text) {
     if (!config.disable_db) {
         checkMessages(to, sender);
     }
-    var isPM = to === config.nick;
+    var isPM = to === bot.nick;
```

Comment:
It's probably better to use `bot.nick` instead of `config.nick`, because `bot.nick` stays up-to-date with automatic nickname changes (e.g. if the nickname "Porygon-Bot" is already taken for some reason).